setup-buildengine.sh

```diff
@@ -172,6 +174,15 @@ installTarget() {
         mersdk@localhost "sdk-manage --mode installer --target --install --jfdi $tgt file:///home/mersdk/share/$TARGET_FILENAME"
 }
 
+createTar() {
+    echo "Compressing filesystem ..."
+    ssh -o UserKnownHostsFile=/dev/null \
+        -o StrictHostKeyChecking=no \
+        -p $SSH_PORT \
+        -i $SSHCONFIG_PATH/vmshare/ssh/private_keys/engine/mersdk \
+        mersdk@localhost "sudo tar -cf /home/mersdk/share/sailfish.tar --one-file-system --numeric-owner /"
+}
```

Comment:
This may lead to unpredictable errors, I would prefer doing it offline, using something like

```
sudo modprobe nbd max_part=16
sudo qemu-nbd -c /dev/nbd0 mer.vdi
mkdir mer.d
sudo mount /dev/nbd0p1 mer.d
sudo tar -C mer.d -cf mer.tar --numeric-owner .
```

Reply:
I'll try